streamly.cabal

```diff
@@ -533,7 +534,7 @@ library
                      , transformers      >= 0.4   && < 0.6
 
                      , heaps             >= 0.3     && < 0.4
-
+                     , filepath          >= 1.4.1 && < 1.5
```

Comment:
I know we have implementeing/describing proper file paths in our agenda. Maybe you can just use `String (FilePath)` for now. This is also fine but we'll be removing this sooner or later.

Reply:
currently, filepath package is used for path concatenation which is platform-independent.